lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1389,6 +1389,16 @@ class IOSTextEditingStrategy extends GloballyPositionedTextEditingStrategy {
   Timer? _positionInputElementTimer;
   static const Duration _delayBeforePlacement = Duration(milliseconds: 100);
 
+  /// The interval is considered as fast between blur subscription and callback.
+  ///
+  /// This is only used for iOS. The blur callback may trigger as soon as
+  /// the creation of the subscription. In that case, the virtual keyboard will
+  /// show and hide again occasionally.
```

Comment:
Nit: Adding "quickly" to make it clear this is weird incorrect behavior.

"
Occasionally in that case, the virtual keyboard will quickly show and hide again.
"